interface/modules/custom_modules/oe-module-weno/src/WenoGlobalConfig.php

```diff
@@ -0,0 +1,130 @@
+        return $GLOBALS[$settingKey] ?? null;
+    }
+
+    public function getGlobalSettingSectionConfiguration()
+    {
+        $settings = [
+            self::WENO_RX_ENABLE => [
+                'title' => 'Enable Weno eRx Service'
+                ,'description' => xl('Enable Weno eRx Service') . ' ' . xl('Contact https://online.wenoexchange.com to sign up for Weno Free eRx service.')
+                ,'type' => GlobalSetting::DATA_TYPE_BOOL
+                ,'default' => ''
+                ,'user_setting' => false
+            ]
+            ,self::WENO_RX_ENABLE_TEST => [
+                'title' => xl('Enable Weno eRx Service Test mode')
+                ,'description' => xl('Enable Weno eRx Service Test mode. This option will automatically include test pharmacies in your pharmacy download')
+                ,'type' => GlobalSetting::DATA_TYPE_BOOL
+                ,'default' => ''
+                ,'user_setting' => false
+            ]
+            ,self::WENO_ENCRYPTION_KEY => [
+                'title' => xl('Weno Encryption Key')
+                ,'description' => xl('Encryption key issued by Weno eRx service on the Weno Developer Page')
+                ,'type' => GlobalSetting::DATA_TYPE_ENCRYPTED
+                ,'default' => ''
+                ,'user_setting' => false
+            ]
+            ,self::WENO_ADMIN_USERNAME => [
+                'title' => xl('Weno Admin Username')
+                ,'description' => xl('This is required for Weno Pharmacy Directory Download in Background Services. Same as email for logging in into Weno')
+                ,'type' => GlobalSetting::DATA_TYPE_TEXT
+                ,'default' => ''
+                ,'user_setting' => false
+            ]
+            ,self::WENO_ADMIN_PASSWORD => [
+                'title' => xl('Weno Admin Password')
+                ,'description' => xl('')
+                ,'type' => GlobalSetting::DATA_TYPE_ENCRYPTED
+                ,'default' => ''
+                ,'user_setting' => false
+            ]
+            ,self::WENO_SETUP_INSTRUCTION_LINK => [
+                'title' => xl('Weno Setup instructions for OpenEMR')
+                ,'description' => xl('Check your Weno Developer page for instructions')
+                ,'type' => GlobalSetting::DATA_TYPE_TEXT
+                ,'default' => ''
+                ,'user_setting' => false
+            ]
+            ,self::WENO_PROVIDER_EMAIL => [
+                'title' => xl('Weno Provider Email')
+                ,'description' => xl('')
+                ,'type' => GlobalSetting::DATA_TYPE_TEXT
+                ,'default' => ''
+                ,'user_setting' => true
+            ]
+            ,self::WENO_PROVIDER_PASSWORD => [
+                'title' => xl('Weno Provider Password')
+                ,'description' => xl('')
+                ,'type' => GlobalSetting::DATA_TYPE_ENCRYPTED
+                ,'default' => ''
+                ,'user_setting' => true
```

Comment:
Doesn't this global also show up in the Administration->Configuration? is that ok?

Reply:
i dont quiet get your question but a section of the global config shows up in the user settings and the admin page. there is a logic in place to handle that.

Reply:
Doesn't this global also show up in the Administration->Configuration? is that ok?